conf/release.yaml

```diff
@@ -86,6 +86,7 @@ excludeFromPackaging:
     - "*~"
     - ".travis.yml"
     - ".appveyor.yml"
+    - "phpstorm.meta.php"
```

Comment:
the file is called ".phpstorm.meta.php" (with a dot in the beginning)

Reply:
i renamed it